public/script.js

```diff
@@ -22,8 +22,19 @@ const DEFAULTS = {
 };
 
 let map;
+const validDates = [];
 /** Main function run to start animation */
 async function main() {
+  await fetch('https://nsidc.org/api/mapservices/NSIDC/wms?service=wms&version=1.1.1&request=GetCapabilities').then(function(response) {
```

Comment:
Look into the XMLHttpRequest (https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) instead for loading the xml file.